workshops/drumpad/README.md

```diff
@@ -0,0 +1,470 @@
+author: '@emmanuel39hanks'
+---
+
+Ever wondered how you can play sounds with code? Well, if yes, then you will love this workshop! We will be creating a drum pad in less than 230 lines of code that plays actual sounds, and if you are a beginner, this won't only be fun and creative but will help you get comfortable with coding!
+
+<p align='center'>
+<img src="https://media.giphy.com/media/CjmvTCZf2U3p09Cn0h/giphy.gif" alt="ready-to-code" >
+</p>
+
+
+# Overview
+
+_Preview of the Drum Pad we are going to be creating_
+
+![Drum Pad Preview](https://cloud-ms6ubs6qu.vercel.app/0screencapture-drum-emmanuel39hanks-repl-co-2020-11-07-23_20_37.png)
+
+Apart from building the drum pad, you will also be learning about different types of events, functions, styling, and more with Vanilla JavaScript, HTML, and CSS. This whole workshop is customizable according to you, and the best part is, it will take less than 20 minutes to complete! 
+
+[Link to demo](https://drum.emmanuel39hanks.repl.co)
+
+## Getting started
+
+Let's start by setting up our IDE (Integrated Development Environment) on Rept.it, a free, online code editor.
+To start your coding right away, navigate to https://repl.it/languages/html, and you will have your whole setup ready when you sign up.
+
+You will see that there are already three files named HTML, CSS, and JavaScript. Navigate to your `index.html` file, and we can start working on the structure of our drum pads in HTML. 
+
+## HTML Part:
+
+We will start writing HTML, going into our `<body>` tag, where we write code that will be shown to the user.
+
+Inside the `<body>` tag, we will create a big header that displays the text `DRUM PAD`, and we will do that with the `<h1>` tag:
+
+```html
+
+<!-- Header tag -->
+<h1>DRUM PAD</h1>
+```
+
+Then just under the `<h1>` tag, we will have three rows and four columns of pads, and each pad will be created with the `<div>` tag. You can think of the `<div>` tag as a box or container, and that's why we are using it here because each of our pads will have a boxy look.
+
+```html
+
+<!-- Parent div -->
+<div>
+
+<!-- nested child divs -->
+  <div>A</div>
+
+  <div>B</div>
+
+  <div>C</div>
+
+  <div>D</div>
+
+  <div>E</div>
+
+  <div>F</div>
+
+  <div>G</div>
+
+  <div>H</div>
+
+  <div>I</div>
+
+  <div>J</div>
+
+  <div>K</div>
+
+  <div>L</div>
+
+</div>
+
+```
+
+To quickly break this down. We have a parent `<div>` tag that nests or holds all of our pads, and then we just pass letters from A to L to easily identify each pad and like I mentioned above, each pad will be given a box look, and that's why we are using `div` tags.
+
+And if we run our code, it should look like this:
+
+![Preview of HTML with no CSS](https://cloud-hqtl5tea3.vercel.app/0screencapture-drumpad-emmanuel39hanks-repl-co-2020-11-03-08_36_46.png)
+
+# CSS Part:
+
+Not so compelling, right? We will now write some CSS to add stlying, which make our drum pad look prettier. Let's do that!
```

Comment:
Worded a bit weirdly, maybe rewrite it as "But this doesn't look that nice, right?"

Reply:
Sure thing